### DISCUSSION TITLE
Win 10 Start Menu Stays On Screen After Closing (Resolved)

### DISCUSSION THREAD

**Discussion by h4rry431** [valinet/ExplorerPatcher#2712] [General]
https://github.com/valinet/ExplorerPatcher/assets/156951184/05256349-94a6-4aec-8832-734142047451

the video is self explanatory.

**Comment by pyrates999** [comment]
Restart File Explorer using the EP properties application when this happens.

https://github.com/valinet/ExplorerPatcher/issues/2711

**Reply by h4rry431** [reply]
tried this and worked! i used winaero tweaker’s restart explorer context menu and worked, thx 😁

**Reply by pyrates999** [reply]
Please close this then.
